scripts/create-launchpad-credentials

```diff
@@ -39,19 +41,25 @@ class AuthorizeRequestTokenRemotely(RequestTokenAuthorizationEngine):
 
 
 def main():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "-l", "--launchpad", dest="launchpad_instance", default="production")
```

Comment:
I've added a bit of documentation.  The process of setting up Launchpad locally typically involves adding `launchpad.dev` etc. to `/etc/hosts`, but a full URI works too if you're doing something even stranger.